banners.yml

```diff
@@ -0,0 +1,19 @@
+banners: 
```

Comment:
Please could we move banners.yml out of the root directory? Maybe if would be a good fit for lib/data? And rename recruitment_banners.yml as per previous comment